jobs/periodic/sync-to-webflow/index.ts

```diff
@@ -0,0 +1,16 @@
+import syncCourses from './sync-courses';
+import { getLogger } from '../../utils/logging';
+import syncLectures from './sync-lectures';
+
+const logger = getLogger();
```

Comment:
Please set a topic, `getLogger('SyncToWebflow')`

Reply:
I'm using the `getLogger` function from the `jobs/utils/logging` file, which doesn't support a topic. Should I rather use the one from log4js?